radixdlt-core/radixdlt/src/main/java/com/radixdlt/store/berkeley/BerkeleyLedgerEntryStore.java

```diff
@@ -363,7 +365,7 @@ private void open() {
 							} else if (substateTypeId == SubstateTypeId.PREPARED_RAKE_UPDATE.id()) {
 								prefixIndexSize = 2 + Long.BYTES;
 							} else if (substateTypeId == SubstateTypeId.VALIDATOR_STAKE_DATA.id()) {
-								prefixIndexSize = 3;
+								prefixIndexSize = 3 + UInt256.BYTES + ECPublicKey.COMPRESSED_BYTES;
```

Comment:
Are we missing the `is_registered` byte?

Reply:
'3' used instead of '2'. I'll add more comments.